pages/docs/reference/sealed-classes.md

```diff
@@ -7,38 +7,86 @@ title: "Sealed Classes"
 
 # Sealed Classes
 
-Sealed classes are used for representing restricted class hierarchies, when a value can have one of the types from a
-limited set, but cannot have any other type. They are, in a sense, an extension of enum classes: the set of values
-for an enum type is also restricted, but each enum constant exists only as a single instance, whereas a subclass
-of a sealed class can have multiple instances which can contain state.
+_Sealed_ classes represent restricted class hierarchies that provide more control over inheritance. 
+All subclasses of a sealed class are known at compile time. No other subclasses may appear after
+a module with the sealed class is compiled. For example, third-party clients can't extend your sealed class in their code.
+Thus, instances of sealed classes can have one of the types from a limited set closed for further extension.
 
-To declare a sealed class, you put the `sealed` modifier before the name of the class. A sealed class can have
-subclasses, but all of them must be declared in the same file as the sealed class itself. (Before Kotlin 1.1,
-the rules were even more strict: classes had to be nested inside the declaration of the sealed class).
+In some sense, sealed classes are similar to enum classes: the set of values
+for an enum type is also restricted, but each enum constant exists only as a _single instance_, whereas a subclass
+of a sealed class can have _multiple_ instances, each with its own state.
+
+To declare a sealed class, put the `sealed` modifier before its name.
 
 <div class="sample" markdown="1" theme="idea" data-highlight-only>
+
 ```kotlin
 sealed class Expr
+
 data class Const(val number: Double) : Expr()
 data class Sum(val e1: Expr, val e2: Expr) : Expr()
 object NotANumber : Expr()
 ```
 </div>
 
-(The example above uses one additional new feature of Kotlin 1.1: the possibility for data classes to extend other
-classes, including sealed classes.)
-
 A sealed class is [abstract](classes.html#abstract-classes) by itself, it cannot be instantiated directly and can have *abstract*{: .keyword } members.
```

Comment:
Do we keep `{: .keyword}` in old docs?

Reply:
Not sure we should care about it right before completely migrating to WH.

Reply:
OK if it works